docs/beman-standard.md

```diff
@@ -235,14 +235,17 @@ must either begin with `detail_` or reside within a subdirectory of
 `include/beman/<short_name>/` called `detail` or begins with `detail_`.
 
 **[DIRECTORY.SOURCES]** RECOMMENDATION: Sources and headers not part of the
-public interface should reside in `src/`.
+public interface should reside in the top-level `src/` directory, and should use
+the same structure from `include/` - e.g., `src/beman/<short_name>/`. Check `CMAKE.AVOID_PASSTHROUGHS`.
```

Comment:
Missing few details here

Reply:
I'm not following the part about avoid passthru? Unfortunately that quick web search doesn't turn up docs on that flag so I think we'd need to say *why* we want that -- aka what it does for us

Reply:
Ah nevermind -- I see what this refers to now.

Reply:
It's a reference to another guideline in the same doc.